.github/workflows/6_build.yml

```diff
@@ -0,0 +1,59 @@
+name: Build extension
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
+      branch:
+        description: 'Branch to run run this workflow on'
+        required: true
+
+jobs:
+  tests:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Checkout branch
+        run: |
+          git checkout ${{ github.event.inputs.branch }}
+      - name: Install Dependencies
+        run: npm install
+      - name: Compile
+        run: npm run compile
+      - name: Replace Readme
+        run: npm run change_readme  ${{ github.event.inputs.npm_channel }}
+      - name: Build vsix 
+        run: cd packages/vscode && npm run package
+      - name: Get tag and asset name
+        id: names
+        run: npm run get_tag_name ${{ github.event.inputs.npm_channel }} ${{ github.event.inputs.extension_version }}
+      - name: create a release
+        id: create_release
+        uses: actions/create-release@v1
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.names.outputs.tag_name }}
+          release_name: ${{ steps.names.outputs.tag_name }}
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ${{github.workspace}}/${{ steps.names.outputs.asset_name }}-${{ github.event.inputs.extension_version }}.vsix
+          asset_name: ${{ steps.names.outputs.asset_name }}-${{ github.event.inputs.extension_version }}.vsix
+          asset_content_type: application/zip     
```

Comment:
Does this content type actually work? It might confuse Github backend.

Reply:
I did try this out on my other repo, works 👍

Reply:
(there is no `vsix` content type unfortunately)

Reply:
Isn't vsix not just a renamed .zip anyway?

Reply:
yes it is